src/list/manage/components/list-sort.js

```diff
@@ -8,32 +8,47 @@ import React from "react";
 
 import styles from "./list-sort.css";
 
+function calcSelectStyle(el) {
+  return 20 + (el.options[el.selectedIndex].text.length * 8) + "px";
+}
+
 export default class ListSort extends React.Component {
   constructor(props) {
     super(props);
     this.state = {
       value: props.sort,
+      selectWidth: "65px",
     };
+    this.selectEl = React.createRef();
```

Comment:
This change is breaking the unit tests.  Not sure what kind of changes are needed in the tests to make it pass, since this is maintaining the right functionality.